extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -307,6 +309,7 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         consentState == CONSENT_POLICY_STATE.UNKNOWN
           ? 1
           : null,
+      'gdpr_consent': consentTuple.consentString,
```

Comment:
Worry this could be quite a long string right?  As such it could cause all of the later parameters to be removed if the url gets truncated.